test/utils/EmacUrlBuilderSpec.scala

```diff
@@ -52,11 +52,38 @@ class EmacUrlBuilderSpec extends SpecBase with MockitoSugar with BeforeAndAfterE
       "not return EMAC URL" in {
 
         when(mockAppConfig.useEmacVatEnrolment).thenReturn(false)
-        when(mockAppConfig.getPortalUrl("vat-change-details-enrol")(Some(vatDecEnrolment))(fakeRequest)).thenReturn("falseUrl")
+        when(mockAppConfig.getPortalUrl("vat-change-details-enrol")(Some(vatDecEnrolment))(fakeRequest)).thenReturn(
+          "falseUrl")
```

Comment:
You have used a full URL above for the test data and `falseUrl` here. Both ways are fine just stay consistent in how you construct test data